news/_posts/translation/2020-09-09-do-you-speak-prestashop-august-2020.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – August 2020 edition"
+subtitle: "Where you'll learn that 3 new languages projects have opened"
+date:  2020-09-09 12:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in August.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+### New manager of the translation projects 
+
+My name is Julie. I joined PrestaShop on September 1st as the new Product Content Manager. I am very happy to start this new adventure and I am looking forward to working together on this translation project! :fire:
+ 
+
+### New projects have been opened!
+
+Would you like to try a new challenge? If you are fluent in one of the languages below, don’t hesitate to contribute. Here are the new projects that have been opened in August:
+Spanish (Costa Rica), French (Belgium), and Maltese.
+
+I would like to thank gcocozza, jmcoget and jbonello who offered to open and help on the three projects. If you would like to start to translate a new language that is not yet available on PrestaShop, feel free to send me an email at translation@prestashop.com. :wink:
+
+
+### Don’t miss the PrestaShop Day Italy, 100% online!
+
+The PrestaShop Day for Italy will take place next 24th September as an online event!
+35 live sessions in Italian are planned with entrepreneurs, partners, tech experts and more. Don’t forget to mark this event on your calendar! :calendar:
+To discover more about this event and to register, I invite you to check this link: https://bit.ly/2CSQlc2
```

Comment:
```suggestion
To discover more about this event and to register, I invite you to [check this link](https://bit.ly/2CSQlc2)
```

This will make it a clickable link 😉